webapp/package.json

```diff
@@ -44,7 +44,7 @@
     "moment": "^2.29.1",
     "mustache": "^4.2.0",
     "ngrx-store-logger": "^0.2.4",
-    "ngx-bootstrap": "^7.0.0",
+    "ngx-bootstrap": "^7.1.2",
```

Comment:
I checked their change logs and then I did a "find" by this library in our code, then I found that we use modals, dropdowns and accordions. Doing a smock test it's probably nice, just to make sure all is okay, by like quickly using the components.
But it appears to be a safe upgrade :)

Reply:
I assume you meant "smoke" test and not "smock test"?  Either way I smoked and smocked it and didn't see any bugs.  Thanks for the note!  Please un-resolve if I missed something.